opae.spec

```diff
@@ -166,15 +142,15 @@ DESTDIR=%{buildroot}  cmake -DCOMPONENT=toolfpgad_api -P cmake_install.cmake
 DESTDIR=%{buildroot}  cmake -DCOMPONENT=toolfpgad_vc -P cmake_install.cmake
 
 prev=$PWD
-pushd %{_topdir}/BUILD/opae/python/opae.admin/
+pushd %{_topdir}/BUILD/%{name}-%{version}/python/opae.admin/
 %{__python3} setup.py install --single-version-externally-managed  --root=%{buildroot} --record=$prev/INSTALLED_OPAE_ADMIN_FILES
 popd
 
```

Comment:
It appears that the packaging directive for opae.io has been removed.  Could you please add it back?

Reply:
OPAE 2.0.0-2 doesn't support  opae.io ,  OPAE.IO  changes are not in OPAE2 .0.0

Reply:
Ah yes, not there for 2.0.  nvm